src/buildCommon.js

```diff
@@ -24,6 +24,13 @@ const mainitLetters = [
     "\\pounds",  // pounds symbol
 ];
 
+const boldsymbolLetters = [
+    "\\imath",   // dotless i
+    "\\jmath",   // dotless j
+    "+",  // plus symbol
+    "-",  // minus symbol
```

Comment:
I looked at the font file for KaTeX_MainBold and realized that there are more symbols in it that we should show in bold, e.g. `=`, `<`, `>`, etc.  We could use `lookupSymbol` to determine whether we have a bold variant or not, that way we can include all of them without the extra table.

Reply:
Agreed; this should ideally not be hard coded.

Reply:
I've removed the hardcoded symbols - this was done for `mathit` so I copied the approach - and am now using lookupSymbol.